lib/models/task.js

```diff
@@ -6,6 +6,17 @@ class Task extends CoreObject {
   run(/*options*/) {
     throw new Error('Task needs to have run() defined.');
   }
+
+  /**
+   * Interrupt comamd with an exit code
+   * Called when the process is interrupted from outside, e.g. CTRL+C or `process.kill()`
+   *
+   * @private
+   * @method onInterrupt
+   */
+  onInterrupt() {
+    process.exit(1);
```

Comment:
nope not yet

note: https://github.com/ember-cli/ember-cli/pull/7019#pullrequestreview-36374109 does tidy things up but does not address ^